### DISCUSSION TITLE
4.1.20-stratego-9 without --print-strategies option

### DISCUSSION THREAD

**Discussion by guiok** [UPPAALModelChecker/UPPAAL-Meta#105] [Q&A] [answered]
Hi,

UPPAAL Stratego 4.1.20 8 and 9 no longer has the --print-strategies option. Is this correct?

How can I do to export the strategies without this option?

Thanks,

**Comment by petergjoel** [comment, marked as answer]
We have recently introduced the following queries to support a more fine-grained export/import:
```
saveStrategy("path.json", S)
strategy S = loadStrategy {a,b,c} -> {x,y,z} ("path.json")
```
Notice that the `loadStrategy` command is currently limited to learned strategies and that the observed variables have to be explicitly stated in `{...} -> {...}` annotation. 

You can see examples of the use in this repeatability package: https://zenodo.org/record/3268381
See specifically the `animation`-subfolder.

**Reply by guiok** [reply]
Thank you. It worked very well.
